CMakeLists.txt

```diff
@@ -281,6 +289,14 @@ endif()
 if(HERMESVM_SANITIZE_HANDLES)
     add_definitions(-DHERMESVM_SANITIZE_HANDLES)
 endif()
+if (HERMES_ENABLE_ADDRESS_SANITIZER)
+    append("-fsanitize=address" CMAKE_CXX_FLAGS CMAKE_C_FLAGS CMAKE_EXE_LINKER_FLAGS)
+endif()
+if (HERMES_ENABLE_UNDEFINED_BEHAVIOR_SANITIZER)
+    add_definitions(-DHERMES_UBSAN)
+    # Do not enable the vptr sanitizer, as it requires RTTI.
+    append("-fsanitize=undefined -fno-sanitize=vptr" CMAKE_CXX_FLAGS CMAKE_C_FLAGS CMAKE_EXE_LINKER_FLAGS)
```

Comment:
As this line is currently with `-fsanitize=undefined`, UBSAN will only print *warnings* to stderr and will not halt the program.

It is probably better to have a harder condition, by using in addition to that flag `-fno-sanitize-recover=all`, which forces the program to halt whenever a UBSAN rule is violated.

Reply:
Done